Dockerfile.dapper

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/submariner/shipyard-dapper-base:0.8.0
+FROM quay.io/submariner/shipyard-dapper-base:release-0.8
```

Comment:
I suggest we dont do this as part of this patch though

Reply:
Yeah, I will make this patch first in the line of the PR, as a separate patch.